dkg/pedersen/mod_test.go

```diff
@@ -150,7 +159,7 @@ func TestPedersen_Scenario(t *testing.T) {
 	for i := 0; i < n; i++ {
 		addr := minogrpc.ParseAddress("127.0.0.1", 0)
 
-		m, err := minogrpc.NewMinogrpc(addr, nil, flat.NewRouter(minogrpc.NewAddressFactory()))
+		m, err := minogrpc.NewMinogrpc(addr, nil, tree.NewRouter(minogrpc.NewAddressFactory()))
```

Comment:
it could be nice (for now) , to keep the two scenari, one with tree routing and one with flat routing.
It shouldn't change a thing and it's the wrong place to test routing, I agree.
On the other hand, this test case actually produced interesting results, so pragmatically it might make sense to keep both (and put a comment saying that some day we should probably get rid of one).